nfs-server/sbin/fix-permissions

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+OWNER=65534
+GROUP=50
+BASEDIR='/nfs-volume/ninebox/kub'
```

Comment:
can we not pull this from an env var?

Reply:
Yes we could. I'll put something in the deploy yml.